cmd/vclusterctl/cmd/list.go

```diff
@@ -146,3 +122,81 @@ func (cmd *ListCmd) Run(cobraCmd *cobra.Command, _ []string) error {
 
 	return nil
 }
+
+func ossToVClusters(vClusters []find.VCluster, currentContext string) []VCluster {
+	var output []VCluster
+	for _, vCluster := range vClusters {
+		vClusterOutput := VCluster{
+			Name:       vCluster.Name,
+			Namespace:  vCluster.Namespace,
+			Created:    vCluster.Created.Time,
+			Version:    vCluster.Version,
+			AgeSeconds: int(time.Since(vCluster.Created.Time).Round(time.Second).Seconds()),
+			Context:    vCluster.Context,
+			Status:     string(vCluster.Status),
+			Pro:        false,
+		}
+		vClusterOutput.Connected = currentContext == find.VClusterContextName(
+			vCluster.Name,
+			vCluster.Namespace,
+			vCluster.Context,
+		)
+		output = append(output, vClusterOutput)
+	}
+	return output
+}
+
+func proToVClusters(vClusters []pro.VirtualClusterInstanceProject, currentContext string) []VCluster {
+	var output []VCluster
+	for _, vCluster := range vClusters {
+		status := string(vCluster.VirtualCluster.Status.Phase)
+		if vCluster.VirtualCluster.DeletionTimestamp != nil {
+			status = "Terminating"
+		} else if status == "" {
+			status = "Pending"
+		}
+
+		context := kubeconfig.VirtualClusterInstanceContextName(vCluster.Project.Name, vCluster.VirtualCluster.Name)
+
+		vClusterOutput := VCluster{
+			Name:       vCluster.VirtualCluster.Name,
+			Namespace:  vCluster.VirtualCluster.Namespace,
+			Context:    context,
+			Connected:  currentContext == context,
+			Created:    vCluster.VirtualCluster.CreationTimestamp.Time,
+			AgeSeconds: int(time.Since(vCluster.VirtualCluster.CreationTimestamp.Time).Round(time.Second).Seconds()),
+			Status:     status,
+			Pro:        true,
+			Version:    vCluster.VirtualCluster.Status.VirtualCluster.HelmRelease.Chart.Version,
```

Comment:
May be empty string in a lot of cases. Should we leave blank, or show `Unknown` or `Latest`?